plainly-plugin/src/ui/App.tsx

```diff
@@ -39,6 +65,36 @@ export function App() {
       )}
       <Sidebar />
       {route}
+      <ConfirmationDialog
+        open={!!showModal}
+        setOpen={setShowNewVersionModal}
+        title="New version available"
+        description={
+          <>
+            A new version of the extension is available! Updating ensures you
+            get the latest features, performance improvements, and critical bug
+            fixes. Check out what's new in the{' '}
+            <ExternalLink
+              to={`https://github.com/plainly-videos/after-effects-plugin/releases/tag/${data?.tag_name}`}
+              text="latest changelog"
+            />{' '}
+            and update now to stay up to date!
+          </>
+        }
+        buttonText="Update now"
+        action={handleLinkClick.bind(
```

Comment:
replace all `.bind()` with inner function, `() => handleLinkClick(https://exchange.adobe.com/apps/cc/202811/plainly-videos')`, that works right?